addons/link_tracker/controller/main.py

```diff
@@ -3,17 +3,25 @@
 
 from odoo import http
 from odoo.http import request
+from odoo.tools.misc import is_a_bot
+from werkzeug.exceptions import NotFound
```

Comment:
Sorry to be picky :/ werkzeug must be before odoo
https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#imports